WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -315,6 +318,24 @@ private extension DashboardViewController {
         navigationItem.title = Localization.title
     }
 
+    func configureShareButton() {
+        // Only show share button if the site is public.
+        guard let site = ServiceLocator.stores.sessionManager.defaultSite, site.isPublic else {
```

Comment:
Nit: Moving this to view model will allow us to unit test this.

Reply:
Updated this in 1ec15ac06953d3812128b3470cc392622928020b 👍